deploy/docker-compose/README.md

```diff
@@ -0,0 +1,390 @@
+	```
+	ERROR: Named volume "$PWD/.md/mdtm:/var/lib/monai:rw" is used in service "task-manager" but no declaration was found in the volumes section. 
+	```
+
+	*WHY? The value of `TASK_MANAGER_DATA` is exposed to the Task Manager container as an environment variable in order for Task Manager to map the volume from the host to the MAP container correctly.*
+
+- The Informatics Gateway service is unhealthy and fails to start with the following error if Docker is not up-to-date:
+  ```bash
+  $ docker logs mdl-ig
+  Failed to create CoreCLR, HRESULT: 0x80070008
+  ```
+  
+## Running a MONAI Deploy Workflow
+
+This package includes Orthanc running and connected to the Informatics Gateway, with all required AE Titles pre-configured.
+
+To get started, download the following DICOM datasets and upload them to Orthanc.
+
+- [Chest CT dataset](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing)
+- [Abdomen CT dataset](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing)
+
+Note: these DICOM datasets were converted to DICOM from Medical Decathlon training and validation mages & may be used for the referenced examples later in this document.
+
+### Upload DICOM Datasets
+
+Navigate to Orthanc via http://localhost:8042, and click *Upload* on the top right-hand corner.
+Drag & drop any DICOM files (no folders) to the blue box on the page and then click *Start the upload*.
+
+Or, upload your files to Orthanc using the *storescu* command from [dcmtk](https://dcmtk.org/dcmtk.php.en).
+```
+storescu -v +r +sd -aec ORTHANC localhost 4242 /path/to/my/dicom/*
+```
+
+Navigate to the home page and click *All studies* to confirm data's been uploaded.
+
+## Sample Workflows
+
+The **sample-workflows** directory includes the following four sample workflow definitions:
+
+- `hello-world.json `: Hello World!
+- `lung-seg.json`: AI Lung Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+- `combo.json`: AI Lung & AI Liver Combo
+
+### Hello World
+
+#### Description
+
+This example uses the `alpine` image to print all files found in the input directory.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
+
+   {"workflow_id":"849d4683-006b-410c-aa17-d0474ee26b7b"}
+   ```
+   If the `curl` command runs successfully, expect a `workflow_id` to be returned and printed to the terminal.
+1. Navigate to Orthanc, select any study and then click *Send to DICOM Modality* from the menu on the left.
+   In the popup dialog, select **MONAI-DEPLOY** to start a C-STORE request to the Informatics Gateway.
```

Comment:
This stayed in "State:Running" for a long time. The logs from the Alpine container just showed these lines ```/var/monai/input/1.2.826.0.1.3680043.2.1125.1.62837601195937447400039784641162299/1.2.826.0.1.3680043.2.1125.1.71161657915368769845404408463327121/1.2.826.0.1.3680043.2.1125.1.30657208734787372391724501851748609.dcm``` 
At this point I am not sure what is happening.